setuptools/msvc.py

```diff
@@ -933,8 +974,11 @@ def SdkTools(self):
         """
         Microsoft Windows SDK Tools
         """
-        bin_dir = 'Bin' if self.vc_ver <= 11.0 else r'Bin\x86'
-        tools = [os.path.join(self.si.WindowsSdkDir, bin_dir)]
+        if self.vc_ver < 15.0:
+            bin_dir = 'Bin' if self.vc_ver <= 11.0 else r'Bin\x86'
+            tools = [os.path.join(self.si.WindowsSdkDir, bin_dir)]
+        else:
+            tools = []
```

Comment:
Oh, I see now, you initialize it empty so you can populate it later. This is getting complicated.